packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -85,12 +96,46 @@ export default class EmailCardDropRequestsRoute {
 
     ctx.type = 'application/vnd.api+json';
 
+    let prepaidCardMarketV2 = await this.cardpay.getSDK('PrepaidCardMarketV2', this.web3.getInstance());
+
+    if (await prepaidCardMarketV2.isPaused()) {
+      return respondWith503(ctx, 'The prepaid card market contract is paused');
+    }
+
+    let quantityAvailable = await prepaidCardMarketV2.getQuantity(cardDropSku);
+
+    let quantityIsBelowNotificationThreshold = quantityAvailable < notifyWhenQuantityBelow;
+    let notificationHasNotBeenSentRecently =
+      !lastNotifiedOnQuantity || lastNotifiedOnQuantity < Date.now() - millisToWaitBeforeNotifying;
+
+    log.trace(
+      `${cardDropSku} has ${quantityAvailable} available, notification threshold is ${notifyWhenQuantityBelow}`
+    );
+
+    if (quantityIsBelowNotificationThreshold && notificationHasNotBeenSentRecently) {
+      Sentry.captureException(
+        new Error(
+          `Prepaid card quantity (${quantityAvailable}) is below cardDrop.email.notifyWhenQuantityBelow threshold of ${notifyWhenQuantityBelow}`
+        ),
+        {
+          tags: {
+            action: 'drop-card',
+            alert: 'web-team',
+          },
+        }
+      );
+
+      lastNotifiedOnQuantity = Date.now();
+    }
```

Comment:
If we're going to be sending the notification by using Sentry alerts, an alternative is: 

1. Set up a Sentry alert that is specifically for having a low quantity of prepaid cards that emails web-team, and people responsible for adding funds (right now everyone who can do this should have access to Sentry as a member)
2. Set an [action interval](https://docs.sentry.io/product/alerts/create-alerts/issue-alert-config/#action-interval-rate-limit) of 30 min to limit the notification-sending there.

This means less state to manage in the hub, and it's easier to change notification interval. I'm not clear if there are any drawbacks to this... are there? I guess having more Sentry issues is one.

Reply:
ya I wish we could manage Sentry alerts in Terraform, I don’t like the untracked-ness of it, but I agree with letting it manage the threshold, thanks.